server/core/server.go

```diff
@@ -74,18 +76,24 @@ type AccountServer struct {
 
 // NewAccountServer creates a new account server with a default logger
 func NewAccountServer() *AccountServer {
-	return &AccountServer{
-		logger: logging.NewNATSLogger(logging.Config{
-			Colors: true,
-			Time:   true,
-			Debug:  true,
-			Trace:  true,
-		}),
+	ac := &AccountServer{
+		logger: NewNilLogger(),
+	}
+	return ac
+}
+
+func (server *AccountServer) ConfigureLogger() natsserver.Logger {
+	opts := server.config.Logging
+	if isWindowsService() {
+		srvlogger.SetSyslogName("NatsAccountServer")
+		return srvlogger.NewSysLogger(opts.Debug, opts.Trace)
+	} else {
+		return srvlogger.NewStdLogger(opts.Time, opts.Debug, opts.Trace, opts.Colors, opts.PID)
```

Comment:
Some linters will tell you that you should not have the `} else {` since previous `if` statement has a return.